wire-ios-automation/ios/src/main/java/com/wearezeta/auto/ios/common/IOSDriverBuilder.java

```diff
@@ -0,0 +1,224 @@
+package com.wearezeta.auto.ios.common;
+
+import com.google.common.collect.ImmutableMap;
+import com.wearezeta.auto.common.Config;
+import com.wearezeta.auto.common.log.ZetaLogger;
+import com.wearezeta.auto.common.usrmgmt.ClientUser;
+
+import java.time.Duration;
+import java.util.logging.Logger;
+
+import io.appium.java_client.ios.IOSDriver;
+import org.json.JSONObject;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.MutableCapabilities;
+import org.openqa.selenium.ScreenOrientation;
+import org.openqa.selenium.WebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class IOSDriverBuilder {
+
+    private static final Logger log = ZetaLogger.getLog(IOSDriverBuilder.class.getSimpleName());
+
+    private URL hubUrl = null;
+    private Capabilities capabilities = null;
+    private MutableCapabilities extraCapabilities = new MutableCapabilities();
+    private List<String> processArgs = new ArrayList<>();
+    private Consumer<String> logListenerHandler = null;
+    private Boolean fullReset = false;
+    private Boolean allowLocation = true;
+    private Boolean allowTouchID = true;
+    private Boolean allowMicrophoneAccess = false;
+    private Boolean allowCameraAccess = false;
+    private Boolean allowAccessToAllPhotos = false;
+    private Boolean allowNotifications = false;
+    private Boolean uninstallAllOtherVersions = false;
+    private static final String[] bundleIDs = {
+            "com.wearezeta.zclient.development",
+            "com.wearezeta.zclient.rc",
+            "com.wearezeta.zclient.ios-release",
+            "com.wearezeta.zclient.internal",
+            "com.wire.ios.bund.columnf.release",
+            "com.wire.bund.ios.c3",
+            "com.wearezeta.zclient.ios-playground",
+            "com.wire.bund.ios.beta.column3",
+            "com.wearezeta.zclient.alpha",
+            "com.wire.bund.ios.beta",
+            "com.wire.bund.ios.column3",
+            "com.wearezeta.zclient.ios.beta"
+    };
```

Comment:
were all these ids public before? + the list looks outdated
I wonder if it's really needed?

Reply:
it could be moved to wire-build-assets if needed